<!DOCTYPE html>
<html>
<head>
	<meta charset="utf-8">
	<link href="./static/css/google_fonts.css" rel="stylesheet" type="text/css">
	<link rel="stylesheet" type="text/css" href="./static/css/main.css">
	<link rel="stylesheet" type="text/css" href="static/css/bootstrap.min.css">
	<title>Random Inferences</title>
</head>
<body>
	<div class="head">
		<img src="./static/images/hero.jpg">
		



	</div>
	<div class="social">
			<h2 class="name">Janu Verma</h2>
			<!-- <h3 class="affiliation">IBM TJ Watson Research Center</h3> -->
		<a href="mailto:j.verma5@gmail.com" target="_blank" class="king-basil">email</a> | <a href="https://twitter.com/januverma" target="_blank" class="king-basil">twitter</a> | <a href="https://github.com/Jverma" target="_blank" class="king-basil">github</a> | <a href="https://www.instagram.com/jverma/" target="_blank" class="king-basil">instagram</a> | <a href="https://www.quora.com/profile/Janu-Verma-2" target="_blank" class="king-basil">quora</a> | <a href="https://www.linkedin.com/in/janu-verma-b79b8823" target="_blank" class="king-basil">Linkedin</a>
	</div>



	<div class="about">
		<p class="main_content">
			<p>
				I'm a machine learning and data visualization researcher. I am currently a senior machine learning scientist at <a href="https://hike.in/">Hike</a> in New Delhi. Previously, I was a researcher in the <a href="http://researcher.ibm.com/researcher/view_group.php?id=6743" target="_blank">HealthCare Analytics Research Group</a> at IBM TJ Watson Research Center in New York. I also worked <a href="http://www.maizegenetics.net/" target="_blank">Buckler Lab @ Cornell University</a> as a Quantitative Researcher, where I worked on the problems at the intersection of machine learning and bioinformatics.
			</p> 
			<p>
				My research interests are in machine learning, data visualization, Human Computer Interaction and computational sciences.
			</p>
			<p>
				I advise start-ups and companies on building and scaling data science and machine learning teams. Email me for consulation and discussion. I'm also available for speaking.
			</p>
			<p>
				I hold a Masters in Theoretical Physics from University of Cambridge in the UK, where I studied string theory and black holes in higher dimensions. I also have an M.S. in pure mathematics (PhD drop-out) from Kansas State University, where I studied algebraic & differential geometry (particularly Donaldson-Thomas Invariants and their wall crossing phenomenon) with Yan Soibelman.
			</p>
			<p>
				I have colaborated with Machine Learning Lab at K-State, have held research positions at Tata Institute of Fundamental Research - Mumbai, and at Jawaharlal Nehru Center for Advanced Scientific Research - Bangalore. 
			</p>
			<p>
				I am a voracious reader and an avid traveler. I hang out at the local coffee shops, which serve as my office away from office.
			</p>
		</p>
	</div>
	<hr>



	<div class="research">
		<h2 class="name1">Research</h2>

		<div class="project_left">
			<h2 class="project_name">Deep Learning on Graphs</h2>
			<img src="./static/images/heterogeneous.png" height="300" width="500">
			<br></br>
			<p class="section">
				An interesting research problem to find a representation of the graph that can be utilized by a machine learning model. Instead of the global summary attributes of the graph, finding a dense representation of the graph that incorporates the local structural information. The idea is to learn a mapping from the graph to a low- dimensional real vector space such that the structural attributes of the graph translate to the geometrical properties in the embedding space. Neural network models have been very succesful at this task. Graph representation learning on heterogeneous networks offers many difficulties, and is not obvious how one can build en embedding that uses the heterogeniety. We propose a method to mine net- work representation that effectively exploits edge heterogeneity in multi-graphs. As an application, we studied the friend recommendation in social networks which is modeled on link prediction in the network.
			</p>
			<div class="paper">Papers
				<ul>
					<li><b>Janu Verma</b>, Srishti Gupta, Deebdoot Mukherjee, Tanmoy Chakraborty <i><a href="https://arxiv.org/abs/1902.03124" target="_blank">Heterogeneous Edge Embeddings for Friend Recommendation</a>,</i> ECIR, April 2019.
				</ul>
			</div>
		</div>
		<hr>


		<div class="project_left">
			<h2 class="project_name">Visual analysis of clustering</h2>
			<img src="./static/images/clustervision.png" height="300" width="500">
			<br></br>
			<p class="section">
				Clustering is a common type of unsupervised machine learning that can be useful for summarizing and aggregating complex multi-dimensional data. However, data can be clustered in many ways, and there exist a large body of algorithms designed to reveal different patterns. In practice, it is quite difficult for data scientists to choose and parameterize algorithms to get the clustering results relevant for their dataset and analytical tasks. To alleviate this problem, we built a clustering analysis system that clusters data using a variety of clustering techniques and parameters and then recommends good clustering results utilizing a variety of quality scoring metrics. Our visual user interface allows users to find high quality clustering results, explore the clusters using a variety of coordinated visualization techniques, and select the cluster result that best suits their task. 
			</p>
			<div class="paper">Papers
				<ul>
					<li><b>Bum Chul Kwon, Ben Eysenbach, Janu Verma, Kenney Ng, Christopher deFilippi, Walter F. Stewart, and Adam Perer </b> <i><a href="" target="_blank">Clustervision: Visual Supervision of Unsupervised Clustering</a>,</i> IEEE Vis, October 2017.
					<li><b>Bum Chul Kwon, Ben Eysenbach, Janu Verma, Kenney Ng, and Adam Perer </b> <i><a href="./static/pdfs/idea2017_clustervision.pdf" target="_blank">Interactive Unsupervised Clustering with Clustervision</a>,</i> KDD IDEA Workshop, August 2017</li>
				</ul>
			</div>
		</div>
		<hr>



		<div class="project_left">
			<h2 class="project_name">Event Sequence Mining and Visualization</h2>
			<img src="./static/images/peekquence-ui.png" height="300" width="500">
			<br></br>
			<p class="section">
				Exploring event sequences in big data is challenging. Though many mining algorithms have been developed to derive the most frequently occurring and the most meaningful sequential patterns, it is yet difficult to make sense of the results. To tackle the problem, we introduce a visual analytics approach - Peekquence, which aims to increase the interpretability of machine learning-based sequence mining algorithms.
			</p>
			<div class="paper">Papers
				<ul>
					<li><b>Bum Chul Kwon, Janu Verma, Adam Perer</b> <i><a href="./static/pdfs/idea2016_peekquence.pdf" target="_blank">Peekquence: Visual Analytics for Event Sequence Data</a>,</i> KDD IDEA Workshop August 2016.
					  <!-- <span class="pdf"><a href="./static/pdfs/idea2016_peekquence.pdf">Read pdf</a></span> </li> -->
					<li><b>Adam Perer, Bum Chul Kwon, Janu Verma </b> <i><a href="./static/pdfs/event2016_peekquence.pdf" target="_blank">The Critical Role of Data Mining for Analyzing Real-World Event Sequences</a>,</i> IEEE Vis Workshop on Temporal and Sequential Event Analysis, October 2016</li>
					<li><b>Bum Chul Kwon, Janu Verma, Adam Perer</b> <i><a href="./static/pdfs/vahc2016_peekquence.pdf" target="_blank">Peeking into Patterns of Clinical Event Sequences with Peekquence</a>,</i> Visual Analytics in HealthCare (VAHC) Novemeber 2016</li>
				</ul>
			</div>
		</div>
		<hr>


		<div class="project_right">
			<h3 class="project_name">Sampling in Visual Analytics</h3>
			<img src="./static/images/samplify.png" height="300" width="500">
			<br></br>
			<p class="section">
				Visual analytics is a powerful tool for exploring and understanding data, as it augments human cognition by leveraging visual perception and facilitates interactive, iterative analysis workflow, which is essential for data experimentation. The visualization of large datasets, however, poses several challenges. First, displaying a large number of data items can create visual clutter, challenging perception and, hence, visual analysis. Here, we make a case for sampling as an essential tool for scalable interactive visual analysis
			</p>
			<div class="paper">Papers
			<ul>
				<li><b>Bum Chul Kwon, Janu Verma, Peter J. Haas, Cagatay Demiralp</b> <i><a href="./static/pdfs/sampling_cga.pdf" target="_blank">Sampling for Scalable Visual Analytics</i></a>, IEEE Computer Graphics and Application, Jan 2017.</li>
			</ul>
		</div>
	</div>
	<hr>

		<div class="project_left">
			<h3 class="project_name">Biological Pathway Analysis</h3>
			<img src="./static/images/biological_path.png" height="300" width="500">
			<br></br>
			<p class="section">
				Biological interpretation and understanding of machine learning based predictive models are highly desirable in healthcare analytics. Predicting Adverse Drug Reactions (ADRs) is extremely important for safe and precision medicine. There are various machine learning based approaches to predict adverse reactions for drugs. These models, though effective, lack biological interpretation and are treated as black-boxes. We propose DrugPathSeeker, a novel in- teractive user interface that integrates the machine learning model, database query API, statistical analysis, and visualization for ex- ploring and understanding of the association between drugs and ADRs.
			</p>
			<div class="papers">Papers
				<ul>
					<li><b>Janu Verma, Heng Luo, Jianying Hu, Ping Zhang</b> <i><a href="" target="_blank">DrugPathSeeker: Interactive UI for Exploring Drug-ADR Relation via Pathways</a>,</i> PacificVis 2017</li>
				</ul>
			</div>
		</div>
		<hr>

		<div class="project_right">
			<h3 class="project_name">Text Classification using word2vec and Convolution Neural Networks</h3>
			<img src="./static/images/ichi.png" height="300" width="500">
			<br></br>
			<p class="section">
			We studied various approaches for categorizing healthcare forum messages into one of seven different categories, based on the message text. We investigated the performance of several standard text classification algorithms on this task. We also explored recently proposed convolution neural network architectures that have been demonstrated to be effective at classifying short texts. We found that an ensemble of the explored models performed better than any of the individual models. Our ensemble was ranked first among the 56 runs submitted to the ICHI Data Challenge 2016.				
			</p>
			<div class="papers">Papers
				<ul>
					<li><b>Janu Verma, Bum Chul Kwon, Yu Cheng, Soumya Ghosh, Kenney Ng</b> <i><a href="./static/pdfs/ichi2016.pdf" target="_blank">Classification of HealthCare Forum Messages</a>,</i> International Conference on HealthCare Informatics (ICHI) 2016</li>
				</ul>
			</div>
		</div>
		<hr>

		<div class="project_left">
			<h3 class="project_name">Weighted Mixed Linear Model for GWAS</h3>
			<img src="./static/images/mlm.png" height="300" width="500">
			<br></br>
			<p class="section">
				Linear mixed models are widely used in humans, animals, and plants to conduct genome-wide association studies (GWAS). Two-stage methods have been proposed to reduce the complexity and increase the computational speed of whole-genome scans. The first stage of the analysis fits raw data to a model including environment and line effects, but no individual marker effects. The second stage involves the whole genome scan of marker tests using summary values for each line as the dependent variable. Missing data and unbalanced experimental designs can result in biased estimates of marker association effects from two-stage analyses. In this study, we developed a weighted two-stage analysis to reduce bias and improve power of GWAS while maintaining the computational efficiency of two-stage analyses. 
			</p>
			<div class="papers">Papers
				<ul>
					<li><b>Shang Xue, Funda Ogut, Zachary Miller, Janu Verma, Peter Bradbury, James Holland</b> <i><a href="http://biorxiv.org/content/early/2017/01/09/099291" target="_blank">Comparison of one-stage and two-stage genome-wide association studies</a>,</i> Submited to Genes|Genomes|Genetics: G3</li>
				</ul>
			</div>
		</div>
		<hr>

		<div class="project_right">
			<h3 class="project_name">Genetic Load in Cassava</h3>
			<img src="./static/images/cassava.png" height="300" width="500">
			<br></br>
			<p class="section">
				Cassava (Manihot esculenta Crantz) is an important staple food crop in Africa and South America, however, ubiquitous deleterious mutations may severely reduce its fitness. To evaluate the deleterious mutations in cassava genome, we constructed a cassava haplotype map by deep sequencing of 241 diverse accessions and identified over 30 million variants. While domestication modified starch and ketone metabolism pathways for human consumption, the concomitant bottleneck and clonal propagation resulted in a large proportion of fixed deleterious amino acid changes, raised the number of deleterious mutations by 24%, and shifted the mutational burden towards common variants. Purging deleterious mutations were ineffective due to the lack of recombination. 
			</p>
			<div class="papers">Papers
				<ul>
					<li><b>Punna Ramu, Williams Esuma, Robert Kawuki, Ismail Rabbi, Chiedozie Egesi, Jessen V Bredeson, Rebecca Bart, Janu Verma, Edward Buckler, Fei Lu</b> <i><a href="https://www.nature.com/ng/journal/vaop/ncurrent/full/ng.3845.html" target="_blank">Cassava haplotype map highlights fixation of deleterious mutations during clonal propagation</a>,</i> Nature Genetics, April 2017</li>
					<li><b>R. Punna, F. Lu, J.Verma, E. Buckler</b> <i>Identifying genetic load by whole genome and transcriptome sequencing of Cassava breading lines,</i> Plant and Animal Genomics XXIII 2015, San Diego, CA</li>
				</ul>
			</div>
		</div>
		<hr>
<!-- 
		<div class="project_left">
			<h3 class="project_name">Cosmological Power Specturm using Machine Learning</h3>
		</div> -->

		<div class="project_right">
			<h3 class="project_name">Lateral Photovoltaic Effect in Organic Semiconductors</h3>
			<img src="./static/images/photovoltaic.png" height="300" width="500">
			<br></br>
			<p class="section">
				We present here a discrete circuit spreading impedance network model for highly disordered semicon- ducting polymeric Schottky devices. Qualitative arguments are presented to correlate the parameters of the spreading function, which represents the network connectivity in the model, to the underlying morphology of the polymer. A detailed theoretical study of the lateral charge transport in organic Schottky devices is carried out using the model. We observe and analyze the dependence of the lateral photovoltage on the polymer morphology, incident power, material specific Schottky junc- tion parameters and the modulation frequency.
			</p>
				<div class="papers">Papers
				<ul>
					<li><b>D. Kabra, J. Verma, N.S. Vidhyadhiraja, K.S. Narayan</b> <i>Model for study of lateral photovolatic effect in ploymeric semiconductors,</i> IEEE Sensors Journal Vol.8, Issue 10 (1663)</li>
					<li><b>J. Verma, D. Kabra, N.S. Vidhyadhiraja, K.S. Narayan</b> <i>A study of charge carrier transport in disordered organic films,</i>MESODIS 2006, Kanpur India</li>
				</ul>
			</div>
		</div>

	</div>
	

	<hr>


	<div class="talks">
		<h2 class="name1">Writings</h2>
		<p class="talk"><i>Mathematical Introduction to Manifold Learning</i><span class="pdf"><a href="https://jverma.github.io/manifold-learning/" target="_blank">Link</a></span></p>

		<p class="talk"><i>Friend Recommendation at Hike</i><span class="pdf"><a href="https://blog.hike.in/networking-d06f58aa3f3e" target="_blank">Link</a></span></p>

		<p class="talk"><i>Linear Regression: Frequentist and Bayesian</i><span class="pdf"><a href="https://medium.com/markovian-labs/linear-regression-frequentist-and-bayesian-447f97c8d330">Link</a></span></p>
	</div> 


	<hr>



	<div class="talks">
		<h2 class="name1">Talks</h2>
		<p class="talk"><i>Beyond QWERTY: Solving the Input Problem of India</i>, <b>Future of Work, YourStory, Bengaluru</b>, Feb 2019 <span class="pdf"><a href="https://yourstory.com/2019/02/hike-replace-keyboard-stickers" target="_blank">post</a></span> </p>
		<p class="talk"><i>Nuts and Bolts of Image Recognition</i>, <b>Software Technology Park of India, Mohali, Chandigarh</b>, Feb 2019 <span class="pdf"><a href="./static/pdfs/eduwaive.pptx" target="_blank">Slides</a></span></p>
			<!-- <span class="pdf"><a href="./static/pdfs/bvp.pdf" target="_blank">Slides</a></span></p> -->
		<p class="talk"><i>ImageNet and Image Recognition</i>, <b>Innvovicon, Bharti Vidya Peeth, New Delhi</b>, Feb 2019 <span class="pdf"><a href="./static/pdfs/bvp.pdf" target="_blank">Slides</a></span></p>
		<p class="talk"><i>Case for Open Data in Machine Learning</i>, <b>Panjab University, Chandigarh</b>, Sept 2018 <span class="pdf"><a href="./static/pdfs/opendata.pdf" target="_blank">Slides</a></span></p>
		<p class="talk"><i>Social Bias in Machine Learning</i>, <b>PyData Delhi Annual Conference 2018</b>, August 2018 <span class="pdf"><a href="./static/pdfs/pydata2018.pdf" target="_blank">Slides</a></span></p>
		<p class="talk"><i>Word Embeddings and NLP</i>, <b>Netaji Subhash Insititute of Technology</b>, April 2018 <span class="pdf"><a href="./static/pdfs/word_embeddings_nsit.pdf" target="_blank">Slides</a></span></p>
		<p class="talk"><i>Data Visualization; Theory and Practices</i>, <b>PyData Delhi Meetup</b>, Feb 2018 <span class="pdf"><a href="./static/pdfs/dataviz_pydata.pdf" target="_blank">Slides</a></span></p>
		<p class="talk"><i>Data Driven Healthcare</i>, <b>Indraprastha Institute of Information Technology (IIIT), New Delhi</b>, Dec 2017 <span class="pdf"><a href="" target="_blank">Slides</a></span></p>
		<p class="talk"><i>Network Visualizations</i>, <b>Annual AICTE Seminar on Machine Learning and HCI, Maharaja Surajmal Institute of Technology, New Delhi</b>, Nov 2017 <span class="pdf"><a href="." target="_blank">Slides</a></span></p>
		<p class="talk"><i>Improving Healthcare with Data Science</i>, <b>Favebook Dev Circle, New Delhi</b>, Oct 2017 <span class="pdf"><a href="./static/pdfs/healthcare.pdf" target="_blank">Slides</a></span></p>
		<p class="talk"><i>Data Science in Healthcare</i>, <b>Cluster Innovation Center, University of Delhi</b>, Sept 2017 <span class="pdf"><a href="./static/pdfs/cic.pdf" target="_blank">Slides</a></span></p>
		<p class="talk"><i>Understanding Clustering: Supervising the Unsupervised</i>, <b>PyData Delhi</b>, Sept 2017 <span class="pdf"><a href="./static/pdfs/PyDataDelhi2017.pdf" target="_blank">Slides</a></span></p>
		<p class="talk"><i>Word Vector Representations (word2vec) and Text Classification,</i> <b>ML-India Gurgaon</b>, Jan 2017, <span class="pdf"><a href="https://github.com/ML-India-Gurgaon-Chapter/Presentations/blob/master/7th%20Meetup%20-%20Janu%20Verma%2C%20IBM%2C%20NY.pdf" target="_blank">Slides</a></span></p>
		<p class="talk"><i>Case for Data Visualization</i>, <b>DataViz Delhi</b>, Nov 2016 <span class="pdf"><a href="./static/pdfs/case_for_viz.key" target="_blank">Slides</a></span></p>
		<p class="talk"><i>Data Science in an Early Start-Up</i>, <b>Round One, New Delhi</b>, Oct 2016</p>
		<p class="talk"><i>Machine learning, data visualization, and software development in computational biology</i>, <b>IBM Research</b>, Sept 2015</p>
		<p class="talk"><i>Machine Learning with Scikit-Learn</i>, <b>Cornell Data Science Club</b>, Sept 2014</p>
		<p class="talk"><i>Author identification in double blind review process</i>, <b>Institute of Biotechnology, Cornell University</b>, Jan 2014</p>
		<p class="talk"><i>Moduli Space of Multi-centered Black Holes and Quiver Representations</i>, <b>University of Kansas</b>, May 2013</p>
		<p class="talk"><i>Motivic Donaldson-Thomas invariants and Kac Conjecture</i>, <b>Institute of Mathematical Sciences, Chennai</b>, Dec 2012</p>
	</div>

	<hr>


	<div class="projects">
		<h2 class="name1">Projects</h2>

		<div class="project_right">
			<h3 class="project_name">Instagram Motifs</h3>
			<img src="./static/images/instagram.jpg" height="200" width="300">
			<br></br>
			<p class="section">Visual analysis of all my Instagram posts using deep convolutional neural networks and 2d tsne visualization.</p>
			<div class="papers">
				<a href="https://jverma.github.io/instagram-motifs/" target="_blank" class="proj_link">Demo</a> | <a href="https://github.com/Jverma/instagram-motifs" target="_blank" class="proj_link">code</a>
			</div>
		</div>
		<hr>

		<div class="project_right">
			<h3 class="project_name">MCMC Play</h3>
			<img src="./static/images/mcmc_play.png" height="300" width="500">
			<br></br>
			<p class="section">An interface for playing with Markov Chain Monte Carlo (MCMC) sampling methods. User can select one of two fitting functions - linear or quadratic. There is a control panel for choosing various parameters. On submission, the system shows two animating windows, one showing sampling in parameter space and other showing the fitting of the curve. More than inspired by tensoflow-play.</p>
			<div class="papers">
				<a href="http://markovianlabs.github.io/MCMC-Play/" target="_blank" class="proj_link">Demo</a> | <a href="https://github.com/markovianlabs/MCMC-Play" target="_blank" class="proj_link">code</a>
			</div>
		</div>
		<hr>



		<div class="project_right">
			<h3 class="project_name">Visa Free</h3>
			<img src="./static/images/visa_free.png" height="300" width="500">
			<br></br>
			<p class="section">Visual representation of the visa network as a Hive Plot layout. The visual shows all visa situations from each country in a source continent to the destinataion continent. The visa types - free, on arrival, required, refused - form an axis in the system. Different colors encodes the visa types. There are various interaction components e.g. highlighting of an edge on hover, click on nodes opens the list of edges emanating from it.</p>
			<div class="papers">
				<a href="https://jverma.github.io/visa_network/" target="_blank" class="proj_link">Demo</a> | <a href="https://github.com/Jverma/visa_network" target="_blank" class="proj_link">Code</a>
			</div>
		</div>
		<hr>


		<div class="project_right">
			<h3 class="project_name">GFF vis</h3>
			<img src="./static/images/gff_vis.png" height="300" width="500">
			<br></br>
			<p class="section">Genome Feature Format (gff) is the standard format for storage and transfer of genomic annotation data, which means positions of chromosomes, genes, RNA etc. on the genome. This visualization uses collapsible tree layout. This captures the hierarchy in the genomic features and the collapsibility helps in save space and focus on portion of concern.</p>
			<div class="papers">
			<a href="http://bl.ocks.org/Jverma/4d8d73ad5f2bb39bab53" target="_blank" class="proj_link">Demo</a> | <a href="https://github.com/Jverma/Visualizing-genome-annotations" target="_blank" class="proj_link">Code</a>
			</div>
		</div>
		<hr>

		<div class="project_right">
			<h3 class="project_name">SEMPS: Cosmological Power Spectra using Machine Learning</h3>
			<img src="./static/images/semps.png" height="300" width="500">
			<br></br>
			<p class="section">
				Built machine learning (gradient boosted trees) models for prediction of power spectra based on the cosmological attributes.
			</p>
			<div class="papers">
				<a href="http://www.ics.uzh.ch/~irshad/semps/" target="_blank" class="proj_link">Demo</a> | <a href="http://arxiv.org/abs/1507.04622" target="_blank" class="proj_link">Read</a>
			</div>
		</div>



	</div>
	<hr>



	<!-- <div class="writing">
		<h2 class="name">Writing</h2>
	</div> -->





	<div class="misc">
		<h2 class="name1">Misc</h2>
		<ul>
			<li><b>TASSEL:</b> TASSEL is a software package (JAVA) to evaluate traits associations, evolutionary patterns, and linkage disequilibrium. I implemented a machine learning based data imputation module, linear algebraic operations (e.g. SVD), weighted mixed linear model for GWAS, and a trie based retrival system for barcodes in GBS. <span class="pdf"><a href="http://www.maizegenetics.net/#!tassel/c17q9" target="_blank">Code</a></span></li>

			<li><b>TextGraphics:</b> Built a python framework for studying a corpus of textual documents using graphical and network analytical methods. Applications include auto-summarization, topic modeling, clustering/communities and link prediction.<span class="pdf"><a href="http://jverma.github.io//TextGraphics/" target="_blank">Code</a></span></li>

			<li><b>Networkx:</b> It is high-level python interface to create D3 JavaScript visualization of graphs. It takes a python networkx graph and renders it into a HTML pages.<span class="pdf"><a href="https://github.com/Jverma/NetworkxD3" target="_blank">Code</a></span></li>

			<li><b>Saransh:</b> An auto text (extractive) summarization webapp. Used text mining techniques like tf-idf vectors, graphical algorithms, and statistical NLP methods.<span class="pdf"><a href="https://www.math.ksu.edu/~jv291/summarize/" target="_blank">Demo</a></span></li>

			<li><b>AdVid:</b> An ad exchange for videos. We used video processing techniques and semantic analysis of the extracted text to suggest relevant video, textual and image content. <b>Big Red Hacks 2014, Cornell University</b> <i>3 prizes</i></li>

			<li><b>Cornell Data Science Club (Project Manager)</b> Advised a team of students on a data science project, and also helped in preparing the education series</li>

			<li><b>StumbleUpon Evergreen Classification Challenge, Kaggle</b> Finished 109/625.</li>
		</ul>
	</div>

</body>